articles/azure/azs-how-create-S2S-VPN.md

```diff
@@ -6,10 +6,10 @@ author: Bailey Lawson
 
 toc_rootlink: Users
 toc_sub1: How To
-toc_sub2:
+toc_sub2: Create a VPN connection
 toc_sub3:
 toc_sub4:
-toc_title: Create a site-to-site connection
+toc_title: Create a site-to-site VPN connection - portal
 toc_fullpath: Users/How To/azs-how-create-S2S-VPN.md

```

Comment:
Add the toc_sub2 to the toc_fullpath, so it becomes:
Users/How To/Create a VPN connection/azs-how-create-S2S-VPN.md

Reply:
Fixed